### DISCUSSION TITLE
How to make it work with arguments

### DISCUSSION THREAD

**Discussion by mzigaib** [DeckCheatz/wemod-launcher#128] [Q&A] [answered]
I am trying to run a game on Steam with the argument ´Steamdeck=1 %command´, I tried several options like ´/home/$USER/wemod-launcher Steamdeck=1 %command´ but it did not work, how can I do that?
Thanks!

**Comment by marvin1099** [comment, marked as answer]
We talked about this in this issue #127.
If anyone alse is looking for this.
